packages/EasyErrorHandler/src/Bridge/Symfony/DependencyInjection/EasyErrorHandlerExtension.php

```diff
@@ -76,6 +78,12 @@ public function load(array $configs, ContainerBuilder $container): void
             BridgeConstantsInterface::PARAM_ERROR_CODES_CATEGORY_SIZE,
             $config['error_codes_category_size']
         );
+        $container->setParameter(
```

Comment:
Why do we need this logic in this class?

Reply:
Not needed anymore. Removed